### PR TITLE
Fortinet FortiManager IPv4 Policy Module

### PR DESCRIPTION
**Pull request by lweighall**
##### SUMMARY

Allows the add/set/delete of IPv4 policies within policy packages on FortiManager. 

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

fmgr_fwpol_ipv4.py
test_fmgr_fwpol_ipv4.py

##### ANSIBLE VERSION

```
ansible 2.5.0 (devel 3bbfbededa) last updated 2017/12/28 11:36:51 (GMT -700)
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/usr/ansible_ftnt/library']
  ansible python module location = /root/ansibleSource/ansible/lib/ansible
  executable location = /root/ansibleSource/ansible/bin/ansible
  python version = 2.7.12 (default, Nov 20 2017, 18:23:56) [GCC 5.4.0 20160609]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/modules/network/fortimanager/fmgr_fwpol_ipv4.py [hunk 1]
```diff
@@ -0,0 +1,1029 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
```
Comment (Akasurde) [inline lib/ansible/modules/network/fortimanager/fmgr_fwpol_ipv4.py]: Use License boilerplate like - 
```
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

File: lib/ansible/modules/network/fortimanager/fmgr_fwpol_ipv4.py [hunk 2]
```diff
@@ -0,0 +1,1029 @@
...
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: fmgr_fwpol_ipv4
+version_added: "2.5"
```
Comment (Akasurde) [inline lib/ansible/modules/network/fortimanager/fmgr_fwpol_ipv4.py]: s/2.5/2.6/

**Comment by maxamillion** [comment]
Hello and thank you for the pull request! I wanted to note that Ansible uses [pytest](https://docs.pytest.org/en/latest/) for it's unit tests as documented [here](https://docs.ansible.com/ansible/devel/dev_guide/testing_units.html#structuring-unit-tests). Please update your unit tests.

**Comment by lweighall** [comment]
Consolidating/Recreating this PR in a few weeks. Much has changed.

**Pull request closed without merging**